### PR TITLE
Add CC-BY-NC-ND-3.0-IGO license and test

### PR DESCRIPTION
**Pull request by vmbrasseur**
Gonna be real happy no longer to need to type that identifier, lemme tell ya.

Closes issue #767

### CODE REVIEW

**Comment by vmbrasseur** [comment]
Travis error:

```
Test for license CC-BY-NC-ND-3.0-IGO failed: Test for license ID CC-BY-NC-ND-3.0-IGO failed due to difference found Normal text of license does not match starting at line #1 column #35 "3" when comparing to template text "
268
269CREATIVE COMMONS CORPORATION IS NOT A LAW FIRM AND DOES NOT PROVIDE LEGAL SERVICES. DISTRIBUTION O...".  Last optional text was not found due to the optional difference: 
270	Normal text of license does not match starting at line #1 column #35 "3" when comparing to template text " Creative Commons Attribution-NonCommercial-NoDerivs 3.0 IGO"
```

**Comment by vmbrasseur** [comment]
Current status: shaving yaks to get `test-one-license` working on my Mac. Was really hoping to put that off for a while longer.

**Comment by vmbrasseur** [comment]
Corrected title in the test text. Now test is passing locally:

```
Pliny:license-list-XML brasseur$ ./test-one-license CC-BY-NC-ND-3.0-IGO
gpg --verify --no-default-keyring --keyring ./goneall.gpg licenseListPublisher-2.1.22.jar.asc
gpg: assuming signed data in 'licenseListPublisher-2.1.22.jar'
gpg: Signature made Tue 15 Oct 18:58:59 2019 PDT
gpg:                using RSA key 8CCBE63DE91BFF1C
gpg: directory '/Users/brasseur/.gnupg' created
gpg: /Users/brasseur/.gnupg/trustdb.gpg: trustdb created
gpg: Good signature from "Gary O'Neall (Key for signing jar files) <gary@sourceauditor.com>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: EA76 0AF7 CBD0 E4F9 013F  06FB 8CCB E63D E91B FF1C
mkdir -p .tmp
java -jar -DLocalFsfFreeJson=false -DlistedLicenseSchema="schema/ListedLicense.xsd" licenseListPublisher-2.1.22.jar LicenseRDFAGenerator src '.tmp' 1.0 2000-01-01 test/simpleTestForGenerator expected-warnings
ERROR StatusLogger No log4j2 configuration file found. Using default configuration: logging only errors to the console. Set system property 'log4j2.debug' to show Log4j2 internal initialization logging.
Processing License List.
Processing Exceptions
Writing table of contents
Completed processing licenses
Pliny:license-list-XML brasseur$
```

**Comment by vmbrasseur** [comment]
Voila.

File: src/CC-BY-NC-ND-3.0-IGO.xml [hunk 1]
```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="CC-BY-NC-ND-3.0-IGO"
+            name="Creative Commons Attribution-NonCommercial-NoDerivs 3.0 IGO" listVersionAdded="3.10">
```
Comment (swinslow) [inline src/CC-BY-NC-ND-3.0-IGO.xml:4]: Just for purposes of aligning with the titles as they currently appear on the license list -- can this be changed to the following?

`Creative Commons Attribution Non Commercial No Derivatives 3.0 IGO`

I recognize that that's a slightly different formatting than the way it appears in the license itself, but I'm guessing for the license list purposes it's probably better to align with the way the others are currently displayed.

File: test/simpleTestForGenerator/CC-BY-NC-ND-3.0-IGO.txt~ [hunk 2]
```diff
@@ -0,0 +1,98 @@
+Attribution-NonCommercial-NoDerivs 3.0 IGO
```
Comment (swinslow) [inline test/simpleTestForGenerator/CC-BY-NC-ND-3.0-IGO.txt~:1]: It looks like the `.txt~` file got included here inadvertently -- can you remove that from the PR?

Comment (vmbrasseur) [reply test/simpleTestForGenerator/CC-BY-NC-ND-3.0-IGO.txt~]: Thanks, Steve! All fixed in the latest commit.

File: src/CC-BY-NC-ND-3.0-IGO.xml (hunk 1 above)
Comment (vmbrasseur) [reply src/CC-BY-NC-ND-3.0-IGO.xml]: All fixed! Thanks for catching that!

**Comment by swinslow** [comment]
LGTM, thanks @vmbrasseur! Merging now.

**Pull request merged**